napari_arboretum/tree.py

```diff
@@ -1,17 +1,19 @@
+"""
+Classes and functions for laying out graphs for visualisation.
+"""
 from dataclasses import dataclass
-from typing import List, Optional, Tuple
+from typing import List, MutableSequence, Optional, Tuple
 
 import numpy as np
-from napari.utils.colormaps import AVAILABLE_COLORMAPS
+
+from .graph import TreeNode
 
 # colormaps
-turbo = AVAILABLE_COLORMAPS["turbo"]
-WHITE = np.array([1, 1, 1, 1], dtype=float)
-RED = np.array([1, 0, 0, 1], dtype=float)
+WHITE = np.array([1.0, 1.0, 1.0, 1.0])
 
 # napari specifies colours as a RGBA tuple in the range [0, 1], so mirror
 # that convention throughout arboretum.
-ColorType = Tuple[float, float, float, float]
+ColorType = MutableSequence[float]
```

Comment:
Guess the only disadvantage here is that we lose the ability to enforce that it is 4 things - but doesn't really matter

Reply:
I looked into this and couldn't find a way of specifying a fixed-length mutable sequence.